commerce_coordinator/apps/core/management/commands/celery.py

```diff
@@ -0,0 +1,20 @@
+import subprocess
+
+from django.core.management.base import BaseCommand
+from django.utils import autoreload
+
+
+def restart_celery():
+    cmd = ('pkill', 'celery')
+    subprocess.call(cmd)
```

Comment:
Just to confirm, is there any chance another celery process could be running that we *don't* want to kill?

Reply:
Good call! I  will try to craft a more targeted command.

Reply:
So, turns out, funny story, that `pkill` command _wasn't actually cleaning up the old processes_ 🤦  The new version _does_ and is pretty specific to the celery instance being managed by the command.